server/views/pdf/createLicenceTaskList.pug

```diff
@@ -33,4 +33,6 @@ block content
     div.paddingTop
       include ../includes/caseListReturn
 
-
+    if reSubmitToDm
+      div.paddingTop
+       include ../includes/backToDm
```

Comment:
should this be removed from here? I thought we were just showing on the task list now?

Reply:
yes. Just forgot to remove it. 
Done